src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "Microsoft.AspNetCore.Hosting": "1.0.0-*",
     "Microsoft.AspNetCore.Server.WebListener": "0.1.0-*",
-    "Microsoft.CodeAnalysis.CSharp": "1.1.0-rc1-20151109-01",
+    "Microsoft.CodeAnalysis.CSharp": "1.2.0-beta1-20160202-02",
```

Comment:
Actually now I take a look, isn't there a lot of redundant transitive dependencies among the projects here?